src/System.Private.CoreLib/shared/System/Threading/WaitHandle.cs

```diff
@@ -78,7 +79,7 @@ public virtual IntPtr Handle
             }
         }
 
-        public SafeWaitHandle SafeWaitHandle
+        public SafeWaitHandle? SafeWaitHandle
```

Comment:
@MadsTorgersen, @jaredpar, we don't have the expressive power to identify that the getter here won't return null but that the setter accepts null.